### ISSUE TITLE
R Workshop: Part 2 2019-06-11

### ISSUE THREAD

**Issue by bpbond** [PNNL-PREMIS/SULI#3]
@stephpenn1 What do you think? A session on RStudio and Git? Or move to `dplyr`?

**Comment by bpbond** [comment]
How about using your blue folders next week: reading flux files into R using `read.csv` and then `readr::read_csv` (reinforce packages); plotting work; and then some basic summarizing using `dplyr`.

**Comment by stephpenn1** [comment]
Yes, I have a handout on dplyr for them too. I think a session on git/github would be more useful once they have a script and dataset they've written on their own and can then practice uploading

**Comment by stephpenn1** [comment]
So for next week, let's make the overarching theme be **packages** and the open source nature of RStudio: showcasing `readr` and `dplyr`

- anyone can build a package (ex: Ben & COSORE!)

**Comment by bpbond** [comment]
I like this, but we'll need to put off functions I think–that'll be too much to cover in a single session.
